packages/ci/src/lib/run.integration.test.ts

```diff
@@ -620,7 +692,7 @@ describe('runInCI', () => {
         diffMdString = await readFile(fixturePaths.diffs.merged.md, 'utf8');
       });
 
-      it('should collect and compare reports for all projects and comment merged diff', async () => {
+      it('event should collect and compare reports for all projects and comment merged diff', async () => {
```

Comment:
This test renaming looks like a mistake, the sentence doesn't make sense:

```suggestion
      it('should collect and compare reports for all projects and comment merged diff', async () => {
```

Reply:
It is, thanks